templates/web/guides/misc/callbacks.md.erb

```diff
@@ -266,6 +267,15 @@ guaranteed to be called even if the thread crashes.
 An example use cases is to ensure that any temporary files are deleted.
 
 
+<a class="anchor" name="mode_changed"></a>
+
+### Mode Changed
+
+#### on_mode_changed
+
```

Comment:
Internal links should be created using this helper:

<%= path "api/Origen/Model.html#current_mode=-instance_method" %>

That means they will work when generating offline, and generally if the root url changes, e.g. if we move to https in future.

Reply:
where does the text go?

Reply:
The above just replaces http://...., so it would be:

`[changes its current mode](<%= ... %>).`